CDP4Composition/Services/GridUpdateService.cs

```diff
@@ -1,19 +1,23 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
+﻿// -------------------------------------------------------------------------------------------------
 // <copyright file="GridUpdateService.cs" company="RHEA System S.A.">
-//   Copyright (c) 2015 RHEA System S.A.
+//   Copyright (c) 2015-2020 RHEA System S.A.
 // </copyright>
-// --------------------------------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------------
 
 namespace CDP4Composition.Services
 {
     using System;
+    using System.Diagnostics.CodeAnalysis;
     using System.Windows;
+
     using DevExpress.Xpf.Grid;
+
     using NLog;
 
     /// <summary>
     /// The service used to lock the update of a grid view when update in the view-model are occuring
     /// </summary>
+    [ExcludeFromCodeCoverage]
```

Comment:
can you write somewhere what the rules are for using the `ExcludeFromCodeCoverage` attribute?